tests/drivers/i2s/i2s_api/src/test_i2s_dir_both_loopback.c

```diff
@@ -18,7 +18,11 @@
 static ZTEST_DMEM const struct device *dev_i2s;
 static ZTEST_DMEM bool dir_both_supported;
 
-void test_i2s_dir_both_transfer_configure_0(void)
+#ifndef TEST_USERMODE
+ZTEST(i2s_dir_both_loopback_test_suite, test_i2s_dir_both_transfer_configure_0)
+#else
+ZTEST_USER(i2s_dir_both_user_loopback_test_suite, test_i2s_dir_both_transfer_configure_0)
+#endif
```

Comment:
I remember @yperess said we just need to write the ZTEST_USER, don't need a #if/#define here.

Reply:
Not sure why this was resolved, yes, you don't need to have the `if/else` here